carsus/io/util.py

```diff
@@ -106,3 +108,39 @@ def read_from_buffer(fname):
     checksum = hashlib.md5(buffer.getbuffer()).hexdigest()
 
     return buffer, checksum
+
+
+def retry_request(
+    url,
+    method,
+    retries=10,
```

Comment:
As I mentioned in #330, hardcoding the retries to 10 may not be the best way to handle the issue.